CONTRIBUTING.md

```diff
@@ -95,6 +95,21 @@ Then add a title and description to your pull request that explains your contrib
 <img width="882" alt="pullr" src="https://user-images.githubusercontent.com/44089458/86092345-fed8c280-baca-11ea-9a04-95991d9c60d2.PNG">  
 - Voila! Your Pull Request has been submitted and will be reviewed by the moderators and merged.🥳
 
+### Note : Do not add images, rather 👇
+
+- We recently have removed all the images and screenshots from our repository and linked them to markdown files.
+    
+    ## How to do that? ##
+
+    - You can do that by hosting all you images and screenshots to any images hosting site such as [imgur](www.imgur.com).
+    - Then link your uploaded images to readme files. An instance 👇
+    
+    ![update_readme](https://media.giphy.com/media/45Bc4gO8j4D0oCLg0g/giphy.gif)
+    
+###### (This helps us save a lot of precious space.) ######

```

Comment:
I don't think we need this line. And as a rule of thumb, we should be using the same Heading Style for every Heading used.